src/lib.rs

```diff
@@ -125,10 +125,32 @@ pub struct StackFuture<'a, T, const STACK_SIZE: usize> {
 }
 
 impl<'a, T, const STACK_SIZE: usize> StackFuture<'a, T, { STACK_SIZE }> {
-    /// Creates a StackFuture from an existing future
+    /// Creates a `StackFuture` from an existing future
     ///
-    /// See the documentation on `StackFuture` for examples of how to use this.
+    /// See the documentation on [`StackFuture`] for examples of how to use this.
+    ///
+    /// Panics if the requested `StackFuture` is not large enough to hold `future` or we cannot
+    /// satisfy the alignment requirements for `F`.
     pub fn from<F>(future: F) -> Self
+    where
+        F: Future<Output = T> + Send + 'a, // the bounds here should match those in the _phantom field
```

Comment:
isn't this technically a semver breaking change 🤔

Reply:
This is just an artifact of how GitHub rendered the diff. These bounds were there before (you can see them at the end of this block of green.